python/ray/_private/gcs_utils.py

```diff
@@ -107,10 +109,14 @@ def get_gcs_address_from_redis(redis) -> str:
     Returns:
         GCS address string.
     """
-    gcs_address = redis.get("GcsServerAddress")
-    if gcs_address is None:
-        raise RuntimeError("Failed to look up gcs address through redis")
-    return gcs_address.decode()
+    count = 0
+    while count < _MAX_GET_GCS_SERVER_ADDRESS_RETRIES:
```

Comment:
should we print some logs here if it failed to fetch gcs address?

Reply:
ok